code/core/scripts/prep.ts

```diff
@@ -306,27 +308,58 @@ async function run() {
         console.log(`compiled ${chalk.cyan(filename)}`);
       });
     } else {
-      await Promise.all(
-        compile.map(async (context, index) => {
-          const out = await context.rebuild();
+      // repo root/bench/esbuild-metafiles/core
+      const metafilesDir = join(__dirname, '..', '..', 'bench', 'esbuild-metafiles', 'core');
+      if (existsSync(metafilesDir)) {
+        await rm(metafilesDir, { recursive: true });
+      }
```

Comment:
Is this right?

Reply:
Yes it's basically just ensureExists && ensureEmpty